k8s/ldata/common/idam/idam-web-admin.yaml

```
@@ -15,7 +15,7 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: idam-web-admin
-    version: 0.2.3
+    version: 0.2.4
```

```suggestion
    version: 0.2.3
```
I guess we want to leave this at 0.2.3